Models/Exceptions/ConflictException.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Monster_Builder_Web_API.Models.Exceptions;
+
+public class ConflictException : Exception
```

Comment:
Added for better error handling, you can do a whole bunch more with custom exceptions but normally they are fairly plain